docs/docusaurus/docs/cloud/expectation_suites/manage_expectation_suites.md

```diff
@@ -24,34 +24,37 @@ Automatically create an Expectation Suite that you can use to determine if your
 
 3. Click **Create New Suite** in the **Expectation Suites** pane.
 
-4. Click **Missingness** and then enter a name for the Expectation Suite in the **Suite name** field.
+4. Click **Automatic (Experimental)**.
 
-5. Click **Generate Expectations**. 
+5. Click **Missingness** and then enter a name for the Expectation Suite in the **Suite name** field.
 
-    It can take several minutes to create the Expectation Suite. When the process is complete, a new Expectation Suite appears in the **Expectation Suites** pane.
+6. Click **Generate Expectations**. 
 
-6. Optional. Run a Validation on the Expectation Suite. See [Run a Validation](/docs/cloud/validations/manage_validations#run-a-validation).
+    It might take several minutes to create the Expectation Suite. When the process is complete, a new Expectation Suite appears in the **Expectation Suites** pane.
 
-## Manually create an empty Expectation Suite 
+7. Optional. Run a Validation on the Expectation Suite. See [Run a Validation](/docs/cloud/validations/manage_validations#run-a-validation).
 
-1. In Jupyter Notebook, run the following code to import the `great_expectations` module and the existing Data Context:
+## Create an empty Expectation Suite
 
-    ```python title="Jupyter Notebook"
-    import great_expectations as gx
-    context = gx.get_context()
-    ```
-2. Run the following code to create an empty Expectation Suite:
+If you have specific business requirements, or you want to examine specific data, you can create an empty Expectation Suite and then add Expectations individually.
 
-    ```python title="Jupyter Notebook"
-    expectation_suite = context.add_expectation_suite(
-    expectation_suite_name="<expectation_suite_name>"
-    )
-    ```
-    Replace `<expectation_suite_name>` with a meaningful name for the Expectation Suite.
+1. In GX Cloud, click **Data Assets** and select a Data Asset in the **Data Assets** list.
+
+2. Click the **Expectations** tab.
+
+3. Click **Create New Suite** in the **Expectation Suites** pane.
+
+4. Click **Manual**.
+
+5. Enter a name for the Expectation Suite in the **Suite name** field.
+
+6. Click **Generate Expectations**. 
+
+    It might take several minutes to create the Expectation Suite. When the process is complete, a new Expectation Suite appears in the **Expectation Suites** pane.
```

Comment:
This line can be removed - when creating an empty suite, it should take almost no time at all.

Reply:
@talagluck, good catch! This text has been removed.